addons/web/controllers/main.py

```diff
@@ -657,6 +657,8 @@ def _get_aggregated_field_names(self):
         """ Return field names of exported field having a group operator """
         aggregated_field_names = []
         for field_name in self._export_field_names:
+            if field_name == '.id':
```

Comment:
I'd think we shouldn't get this in a first place, no? Why do we send `.id`?

Reply:
In the export/import modules '.id' and 'id' are referencing fields.
'.id'  is used for the Database ID
and 'id' is used for External ID.

The problem here is that the group don't have .id but the lines exported has, and we cannot remove it from the list of fields because the groups will have a different number of columns that the lines when exporting.

Reply:
what should be changed here?